ocpmodels/trainers/dist_forces_trainer.py

```diff
@@ -304,18 +305,60 @@ def train(self):
 
                 iters += 1
 
-                # Evaluate val set. -1 for after each epoch.
-                eval_every = self.config["optim"].get("eval_every", -1)
-                if iters % eval_every == 0 and eval_every != -1:
+                # Evaluate on val set every `eval_every` iterations.
+                if iters % eval_every == 0:
```

Comment:
Don't we still need to make sure eval_every `!= -1`? `iters % -1` is always 0 otherwise, and therefore validated at each parameter update.

Reply:
Yup, makes sense, my bad